backend/graphql/types/testType.ts

```diff
@@ -88,18 +100,24 @@ const testType = gql`
   type TestResponseDTO {
     id: ID!
     name: String!
-    duration: Int!
     admin: UserDTO!
     questions: [[QuestionComponent]]!
     grade: Int!
+    assessmentType: AssessmentType!
+    curriculumCountry: String!
+    curriculumRegion: String!
+    Status: Status!
```

Comment:
```suggestion
    assessmentType: AssessmentTypeEnum!
    curriculumCountry: String!
    curriculumRegion: String!
    Status: StatusEnum!
```
can we rename these for consistency with other files?